espresso-tests/.env

```diff
@@ -1,21 +1,23 @@
-# env vars for chain name and rpc_url
-CHAIN_NAME="1337"
-L2_CHAIN_NAME="412346"
-RPC_URL="http://localhost:8545"
-L2_RPC_URL="http://localhost:8547"
-# env vars for new OSP deployment
+# Environment variables for chain name and rpc_url
+# These are essential for the upgrade
+PARENT_CHAIN_CHAIN_ID="1337"
+CHILD_CHAIN_CHAIN_NAME="412346"
```

Comment:
I there a reason to call this "name"? It's also the chain ID.

Reply:
No, we can rename this. In my head chain name and chain id are interchangeable which is not the case for Arbitrum or Ethereum.